src/Chirp.Razor/wwwroot/scripts/mode.js

```diff
@@ -0,0 +1,15 @@
+﻿window.onload = function() {
+    let savedTheme = localStorage.getItem('theme');
+    if (savedTheme) {
+    document.getElementById('themeStyle').href = savedTheme;
+    }
+}
+
+document.getElementById('toggleMode').addEventListener('click', function () {
+    var link = document.getElementById('themeStyle');
+    var newTheme = link.href.includes('lightmode.css') ? '/css/style.css' : '/css/lightmode.css';
```

Comment:
I recommend that we use variables instead of injecting a mirrored css with other colors. Much harder to maintain.
https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties

Reply:
Do commits 0c5c2f2ae90bbbd9f5039c53ce68978edc29f85d, 051875737a5991dacdea1452f1c455aac24d8b01 solve this?